techbuzz/src/main/java/com/sivalabs/techbuzz/notifications/JavaEmailService.java

```diff
@@ -33,7 +34,7 @@ public void sendEmail(String template, Map<String, Object> params, String to, St
             MimeMessage mimeMessage = emailSender.createMimeMessage();
             MimeMessageHelper helper = new MimeMessageHelper(mimeMessage, "utf-8");
             helper.setFrom(properties.adminEmail());
-            helper.setTo(to);
+            helper.setTo(InternetAddress.parse(to));
```

Comment:
Usually these Weekly NewsLetter type of emails will be sent individually instead of one email to all of them, especially in "to" address. To keep it simple, we can include the email addresses in "Bcc" and send the weekly updates in one email.

Reply:
I have made the changes. I was not able to test Sendgrid since I don't have Sendgrid api key